worker/stripe_charge_succeeded.js

```diff
@@ -11,10 +30,14 @@ module.exports = function(notifier_messager, mailroom) {
     }
 
     var currency_code = event.data.currency.toUpperCase();
+    // If this is a zero decimal currency then use it directly
+    // Otherwise divide by 100 to get currency fractional value
+    var amount = zeroDecimalCurrencies.indexOf(currency_code) !== -1 ?
+      amount : (event.data.amount / 100).toFixed(2);
```

Comment:
Saying it's here because we're just being safe is fine and I'll say we can leave it. if you want to remove it though I would be ok with that too.